UPGRADE-6.3.md

```diff
@@ -392,6 +392,12 @@ From a technical point no changes are required if the project is a shop system.
 should consider removing menu categories that are nevertheless created during migration. Note that there will be some
 error messages during migration that complain about missing shop tables and modules in this case, which can be ignored. 
 
+## Home Page Changes
+
+As the main menu is now displayed in the sidebar, the classic main menu was replaced by a welcome screen (which we plan
+to replace by a dashboard in a future Chameleon release). If users would like to keep the classic main menu yet, it can
+be restored by setting the config value `chameleon_system_core: backend: main_pagedef: 'main'` in `config.yml`.
```

Comment:
Is "main_page" and "main" sufficiently clear here?

At the moment I don't know what this is (or where).

Reply:
I suppose it should be clear enough once the reader knows how pagedefs work in the backend (which should also be documented). Otherwise they should just follow this instruction - I think in this place we can't add meaningful explanation without also explaining the whole pagedef concept. Or do you have something in mind?

Reply:
Parts of it I understood now: the "home page" is the main area which used to show the 3-column menu.

What is the default here (the other possibilities instead of "main")?
main has here the meaning of "classic menu"?

This is actually a pagedef?

If I specify "main" here what will happen to the side bar menu?

Reply:
Yes, "main" is the pagedef that contains the classic main menu. Default value is the new "welcome" pagedef; this can be viewed with `app/console config:dump-reference ChameleonSystemCoreBundle`, so I'm against duplicating this piece of documentation. In principle any backend pagedef can be used, so there isn't a complete list (some pagedefs wouldn't work, as they require GET parameters, e.g. the table editor needs a table ID; I'm quite sure the user doesn't want to configure any arbitrary pagedef but is aware if a pagedef can be used without further configuration, so I think it's OK as is).

The sidebar is also visible on the `main` page, so there are two menus (which is the reason for this PR). The sidebar should be visible on any page, especially as it will be the only menu from a future Chameleon version on and users should get used to it.

Reply:
ok.

Then only two details still:
Shouldn't the view be named "classic_menu" or similar?

And one could document that any view can be specified here besides "main" if desired.

Reply:
I'd leave it unchanged as it is "deprecated" and as some users will even be more familiar with `main`. Most users won't need this anymore, others only need to perform this one instruction - even if it isn't perfectly clear why it's called `main`, it will just work and doesn't need further thoughts by users.